yellowbox_snowglobe/session.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, owner: SnowGlobeAPI, db: Optional[str], schema: str):
         self.owner = owner
 
         self.token = str(self.next_token)
-        self.next_token += 1
+        SnowGlobeSession.next_token += 1
```

Comment:
```suggestion
        type(self).next_token += 1
```

Reply:
why?
for compatibility with inherited classes?

Reply:
yes